### PR TITLE
Extend the tests and coverage of group_param_types checker

### PR DESCRIPTION
**Pull request by s2gatev**

### CODE REVIEW

File: checker/group_param_types_checker.go [hunk 1]
```diff
@@ -84,6 +90,27 @@ func typeName(expr ast.Expr) string {
 		return result + " " + typeName(expr.Value)
 	case *ast.Ellipsis:
 		return "..." + typeName(expr.Elt)
+	case *ast.MapType:
+		return fmt.Sprintf("map[%s]%s", typeName(expr.Key), typeName(expr.Value))
+	case *ast.ArrayType:
+		return fmt.Sprintf("[%s]%s", typeName(expr.Len), typeName(expr.Elt))
+	case *ast.FuncType:
+		var paramTypes []string
```
Comment (catiepg) [inline checker/group_param_types_checker.go]: Maybe we can move this case to a separate function?

Comment (s2gatev) [reply checker/group_param_types_checker.go]: Done.

File: lingo.yml [hunk 2]
```diff
@@ -30,6 +30,6 @@ checkers:
     tab_width: 4
   left_quantifiers:
   func_cyclo:
-    max: 15
+    max: 20
```
Comment (catiepg) [inline lingo.yml]: Why do we need to increase the cyclomatic complexity?

Comment (s2gatev) [reply lingo.yml]: https://github.com/s2gatev/lingo/pull/24/files#diff-be28c70f8d6da8565c996817f981df07R68

Comment (catiepg) [reply lingo.yml]: Since you extracted the `funcTypeName` helper, I believe the file is still within the 15.

Comment (s2gatev) [reply lingo.yml]: Yes, updated.

**Comment by s2gatev** [comment]
I plan to deprecate the doc/checkers.md file and somehow use the guide command or something similar to generate it.

**Pull request merged**